### PR TITLE
added friends list and friend avatars

### PR DESCRIPTION
**Pull request by Kevbyte**

### CODE REVIEW

**Comment by Kevbyte** [comment]
lgtm

File: client/app/battle/battle.js [hunk 1]
```diff
@@ -81,6 +82,8 @@ angular.module('battlescript.battle', [])
         $scope.longestStreak = stats.longestStreak;
         $scope.totalWins = stats.totalWins;
         $scope.totalPoints = stats.totalPoints;
+        $scope.avatar = stats.avatar;
+        console.log("$scope.avatar", $scope.avatar);
```
Comment (codydaig) [inline client/app/battle/battle.js]: console.log();

File: client/app/dashboard/dashboard.html [hunk 2]
```diff
@@ -72,11 +68,12 @@ <h3 class="dashboard-stats-entry__title">All Time</h3>
               </div>
             </div>
           </div>
-          <div class="grid__item grid__item--20-l">
+          
+
```
Comment (codydaig) [inline client/app/dashboard/dashboard.html]: Extra White Space

File: client/app/dashboard/dashboard.js [hunk 3]
```diff
@@ -87,22 +87,50 @@ angular.module('battlescript.dashboard', [])
...
   ////////////////////////////////////////////////////////////
-  $scope.friends = {cody: 'daig', andrew: 'kishino', lina: 'lu'};
+  // $scope.friends = {cody: 'daig', andrew: 'kishino', lina: 'lu'};
+  $scope.friends = {};
+
+  $scope.fetchFriends = function() {
+    Users.getFriends($scope.username).then(function(friends) {
+      _.each(friends.data.friends, function(friend) {
+        Users.getStats(friend)
+              .then(function(stats){
```
Comment (codydaig) [inline client/app/dashboard/dashboard.js]: Indentation

File: client/app/dashboard/dashboard.js [hunk 4]
```diff
@@ -129,13 +157,30 @@ angular.module('battlescript.dashboard', [])
...
+    'name': 'Cat3'
+    }, {
+    'img': 'http://40.media.tumblr.com/e6ae65fc3bda818280973566e515e3f5/tumblr_nq4pe4th2i1tvc2voo1_500.png',
+    'name': 'Cat4'
+    }, {
+    'img': 'http://38.media.tumblr.com/2f1c6252cc6c2271fef495328f3698ba/tumblr_inline_mpqrajFx911qz4rgp.png',
+    'name': 'Cat5'
     }
   ];
+
```
Comment (codydaig) [inline client/app/dashboard/dashboard.js:182]: Extra White Space

File: client/app/dashboard/dashboard.js [hunk 5]
```diff
@@ -151,10 +196,13 @@ angular.module('battlescript.dashboard', [])
...
     Users.avatarChange($scope.username, $scope.my.favorite.img)
-      .then(function() {
+      .then(function(res) {
+        // console.log("setAvatar res.data", res.data)
+        $scope.getStats($scope.username);
+        $scope.toShowImagePreview = false;
     });
-    $scope.toShowImagePreview = false;
-    $scope.getStats($scope.username);
+
```
Comment (codydaig) [inline client/app/dashboard/dashboard.js]: Extra White Space

File: client/assets/css/dashboard.css [hunk 6]
```diff
@@ -5,6 +12,7 @@
             flex: 0 0 20%;
 };
 
+
```
Comment (codydaig) [inline client/assets/css/dashboard.css:15]: Extra White Space

File: client/assets/css/newstyle.css [hunk 7]
```diff
@@ -1270,3 +1276,15 @@ button:focus {
...
+  max-width: 50px;
+  height: auto;
+  margin-bottom: -15px;
+}
+
+.friendavatar {
+  max-width: 50px;
+  height: auto;
+  margin-bottom: -15px;
+}
```
Comment (codydaig) [inline client/assets/css/newstyle.css:1290]: Semi Colon

**Comment by AndrewKishino** [comment]
lgtm

**Comment by codydaig** [comment]
LGTM

**Pull request merged**